index.html

```diff
@@ -7663,64 +7659,276 @@ <h2>Acknowledgements</h2>
 The Working Group thanks the following individuals not only for their
 contributions toward the content of this document, but also for yeoman's work
 in this standards community that drove changes, discussion, and consensus among
-a sea of varied opinions: Matt Stone, Gregg Kellogg, Ted Thibodeau Jr, Oliver
-Terbu, Joe Andrieu, David I. Lehn, Matthew Collier, and Adrian Gropper.
+a sea of varied opinions: David Chadwick, Dave Longley,
+Ted Thibodeau Jr., Brent Zundel, Ivan Herman, and Joe Andrieu.
```

Comment:
```suggestion
a sea of varied opinions: Brent Zundel, Dave Longley, David Chadwick,
Gabe Cohen, Ivan Herman, Joe Andrieu, and Ted Thibodeau Jr.
```

Alphabetize by first name and add @decentralgabe, in light of his leadership in conceiving of, socializing, and obtaining consensus to pass the Miami resolution, which gave us a "big-tent" model of Verifiable Credentials.  Gabe was also instrumental in negotiations enabling us to consistently use media type names based on "vc" and "vp" across the family of specifications.

Reply:
The list is sorted in order of the count and volume of comments the individual has contributed. I have no objection to adding Gabe if the group feels that he doesn't have to meet the same requirements of the others in the list.

Reply:
What are the requirements that you are referring to, @msporny?  They are currently unstated.

Reply:
A comment score greater than 500 for this specification:

https://github.com/w3c/vc-data-model/pull/1579#issue-2782821280

Reply:
Leadership in a working group is manifested in many ways not reflected in the scores.  Both of the demonstrations of leadership I cited motivating @decentralgabe's inclusion in this list (the Miami resolution, the media type negotiations) are not captured by these scores, but both were significant to the progress of this specification and the consensus behind it.  Gabe clearly meets the bar of performing "yeoman's work in this standards community that drove changes, discussion, and consensus among a sea of varied opinions".

Also, I'll note that a cursory look at the data posted demonstrates that the methodology used is significantly flawed.  For instance, Tony Nadalin had a higher score than many significant contributors to the 2.0 effort, even though he barely participated in 2.0, if at all.  (Please do not interpret this remark as a slight on Tony, who I respect.)  It is simply a concrete example of the data used to generate these scores not matching what actually occurred.